azure-pipelines-integration.yml

```diff
@@ -1,14 +1,14 @@
-phases:
-- phase: Windows_VisualStudio_Integration_Tests
-  queue:
-    name: dotnet-external-temp-vs2017
-    timeoutInMinutes: 90
-    parallel: 2
+jobs:
+- job: Windows_VisualStudio_Integration_Tests
+  pool: dotnet-external-temp-vs2017
+  strategy:
+    maxParallel: 2
```

Comment:
Is defining "maxParallel" necessary?

Reply:
Curious about this as well. I would assume that parallel would be implicitly the number of job entries in a jobs element. 

---
In reply to: [238359120](https://github.com/dotnet/roslyn/pull/31498#discussion_r238359120) [](ancestors = 238359120)

Reply:
Yeah, maxParallel is only useful if you want to ratchet *down* the number of jobs you spin up at a time. I left it because the existing file had it.